packages/trackerless-network/src/logic/DiscoveryLayerNode.ts

```diff
@@ -1,4 +1,13 @@
-import { DhtAddress, PeerDescriptor, RingContacts } from '@streamr/dht'
+import { DhtAddress, PeerDescriptor, RingContacts, ServiceID } from '@streamr/dht'
+import { StreamPartID } from '@streamr/utils'
+
+export const DEFAULT_DISCOVERY_LAYER_KBUCKET_SIZE = 4
```

Comment:
https://github.com/streamr-dev/network/blob/main/developer-docs/index.md#naming-counts

```suggestion
export const DEFAULT_DISCOVERY_LAYER_KBUCKET_NODE_COUNT = 4
```

At some point we refactored `PeerManager` so that it at least partially hides the `KBucket` terminology, and uses higher level concepts instead (e.g. neighbors, nearby contacts, connections). Does this config option control any of those higher level concepts, or should we keep this name?

Reply:
This is used specifically to control the sizes of the k-buckets that the neighbors are placed in. So it's specifically used to tweak the behaviour of kademlia's k-buckets.  Simply calling it neighbor-something might be a bit challenging because of that.